src/routes/safe/components/Transactions/TxsTable/columns.js

```diff
@@ -34,29 +33,40 @@ type TxData = {
 
 export const formatDate = (date: string): string => format(parseISO(date), 'MMM d, yyyy - HH:mm:ss')
 
-export const getIncomingTxAmount = (tx: IncomingTransaction) => {
-  const txAmount = tx.value ? `${new BigNumber(tx.value).div(`1e${tx.decimals}`).toFixed()}` : 'n/a'
-  return `${txAmount} ${tx.symbol || 'n/a'}`
+type TxValues = {
+  value?: string | number,
+  decimals?: string | number,
+  symbol?: string,
 }
 
-export const getTxAmount = (tx: Transaction) => {
-  const web3 = getWeb3()
-  const { fromWei, toBN } = web3.utils
-
-  let txAmount = 'n/a'
-
-  if (tx.isTokenTransfer && tx.decodedParams) {
-    const tokenDecimals = tx.decimals.toNumber ? tx.decimals.toNumber() : tx.decimals
-    txAmount = `${formatAmount(
-      BigNumber(tx.decodedParams.value)
-        .div(10 ** tokenDecimals)
-        .toString(),
-    )} ${tx.symbol}`
-  } else if (Number(tx.value) > 0) {
-    txAmount = `${fromWei(toBN(tx.value), 'ether')} ${tx.symbol}`
+const NOT_AVAILABLE = 'n/a'
+
+const getAmountWithSymbol = ({ decimals = 0, symbol = NOT_AVAILABLE, value }: TxValues, useFormatAmount = false) => {
```

Comment:
I would rename `useFormatAmount` to `precise` because this is what it means: e.g if we want a precise amount or not

Reply:
also if we ever change the name of `formatAmount` function the name will be obsolete :P

Reply:
done! https://github.com/gnosis/safe-react/pull/812/commits/10e8a977489fd8ccb6fb68cfe9b9193d67d767cd

Reply:
Left a comment

Reply:
yes... :facepalm: what do you think about `usePrecise`? So implementation/usage remains unchanged

Reply:
tbh I don't like really the usage of `use` :D imo it's weird to use such name for something ot her than hooks in react app codebase

To keep the implementation the same you may use `formatted`  as a name

Reply:
There https://github.com/gnosis/safe-react/pull/812/commits/61c54cedc6ee10f95a4ce0437ad9b27d54d00f9b